### PR TITLE
Fix: Interchange initialises own triggers - closes #11403

### LINKED ISSUE #11403: [Interchange] necessity of the utility library triggers is missing in Interchange docs
I was experiencing some issues with Interchange and Foundation 6.4.3. 
On resize, the images set in the arrays didn't get replaced. I did include all the libraries as mentioned in the docs below the 'JavaScript Reference' section. 

When I started debugging in the foundation.interchange.js file, I noticed that the event resizeme.zf.triggers was called. After including the foundation.util.triggers.js file, interchange worked as it should.

Please add foundation.util.triggers.js to the 'JavaScript Reference' section of the docs of Interchange to make sure it's up to date again.

### PR DESCRIPTION
**Pull request by MilesStanton**
Resolve issue where resizeme event will not trigger without other plugins, because triggers are never initialised.

## Description

...

- Closes 

## Types of changes

- [ ] Documentation
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (anything that would change an existing functionality)
- [ ] Maintenance (refactor, code cleaning, development tools...)

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] The pull request title and template are correctly filled.
- [x] The pull request targets the right branch (`develop` or `develop-v...`).
- [x] My commits are correctly titled and contain all relevant information.
- [x] I have updated the documentation accordingly to my changes (if relevant).
- [x] I have added tests to cover my changes (if relevant).

### CODE REVIEW

**Comment by MilesStanton** [comment]
Relates to https://github.com/foundation/foundation-sites/issues/11907

File: js/foundation.interchange.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { MediaQuery } from './foundation.util.mediaQuery';
 import { Plugin } from './foundation.core.plugin';
 import { GetYoDigits } from './foundation.core.utils';
 
+import { Triggers } from './foundation.util.triggers';
```
Comment (DanielRuf) [inline js/foundation.interchange.js]: Can you remove the empty line?

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: js/foundation.interchange.js (hunk 1 above)
Comment (MilesStanton) [reply js/foundation.interchange.js]: Sure can, only put it in because it was spaced like this in the other plugins

**Comment by DanielRuf** [comment]
> Relates to #11907

Isn't this a duplicate then? =)
I think one issue is enough.

**Comment by MilesStanton** [comment]
I may have misunderstood the process, the only other issue I could find was requesting a documentation change, so I made the related issue to address this, and then made the pull request to fix the issue. If this repo doesnt work like that then my apologies, please feel free to close the issue if theres already one that it can use

**Review by DanielRuf — APPROVED** [review]

**Comment by joeworkman** [comment]
This pull request has been mentioned on **Foundation Open Source Community**. There might be relevant details there:

https://foundation.discourse.group/t/foundation-for-sites-6-6-2/1936/1

**Pull request merged**